packages/material-ui/src/Badge/Badge.js

```diff
@@ -129,20 +129,20 @@ Badge.propTypes = {
   /**
    * The badge will be added relative to this node.
    */
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
   /**
    * Override or extend the styles applied to the component.
    * See [CSS API](#css) below for more details.
    */
-  classes: PropTypes.object.isRequired,
+  classes: PropTypes.object,
   /**
    * @ignore
    */
   className: PropTypes.string,
   /**
    * The color of the component. It supports those theme colors that make sense for this component.
    */
-  color: PropTypes.oneOf(['default', 'primary', 'secondary', 'error']),
+  color: PropTypes.oneOf(['default', 'error', 'inherit', 'primary', 'secondary']),
```

Comment:
Conflict TypeScript specifies that `inherit` is an option

Reply:
Doesn't look like it. Seems like you spotted an out-of-date type declaration :+1:

Reply:
Removed